libfwupdplugin/fu-i2c-device.h

```diff
@@ -22,7 +22,7 @@ struct _FuI2cDeviceClass {
  *
  * Since: 2.0.0
```

Comment:
this should bump now

Reply:
I'm not so sure; the `Since` is really useful to plugin authors to know "when did this get added" -- much less useful is "what changed the ABI" -- given you can still use it as `fu_device_has_private_flag(device, FU_I2C_DEVICE_PRIVATE_FLAG_NO_HWID_GUIDS)` without any changes to the plugin.